CSA.md

```diff
@@ -1,215 +1,221 @@
 
-    6.3 <span class="header_3">Termination.</span>  Either party may terminate this Agreement if the other party (a) fails to cure a material breach of the Agreement within 30 days after receiving notice of the breach; (b) materially breaches the Agreement in a manner that cannot be cured; (c) dissolves or stops conducting business without a successor; (d) makes an assignment for the benefit of creditors; or (e) becomes the debtor in insolvency, receivership, or bankruptcy proceedings that continue for more than 60 days. In addition, either party may terminate an affected Order Form if a Force Majeure Event prevents the Cloud Service from materially operating for 30 or more consecutive days, and <span class="coverpage_link">Provider</span> will pay to <span class="coverpage_link">Customer</span> a prorated refund of prepaid fees for the remainder of the <span class="orderform_link">Subscription Period</span>. A party must notify the other of its reason for termination.
+    6.3 <span class="header_3">Termination.</span>  Either party may terminate this Agreement if the other party (a) fails to cure a material breach of the Agreement within 30 days after receiving notice of the breach; (b) materially breaches the Agreement in a manner that cannot be cured; (c) dissolves or stops conducting business without a successor; (d) makes an assignment for the benefit of creditors; or (e) becomes the debtor in insolvency, receivership, or bankruptcy proceedings that continue for more than 60 days. In addition, either party may terminate an affected Order Form if a Force Majeure Event prevents the Product from materially operating for 30 or more consecutive days, and <span class="keyterms_link">Provider</span> will pay to <span class="keyterms_link">Customer</span> a prorated refund of prepaid fees for the remainder of the <span class="orderform_link">Subscription Period</span>. A party must notify the other of its reason for termination.
 
     6.4 <span class="header_3">Effect of Termination.</span>  Termination of the Agreement will automatically terminate all Order Forms. Upon expiration or termination:  
 
-      (a) <span class="coverpage_link">Customer</span> will no longer have any right to use the Cloud Service, <span class="orderform_link">Product Support</span>, or <span class="orderform_link">Professional Services</span>.
+      (a) <span class="keyterms_link">Customer</span> will no longer have any right to use the Product, <span class="orderform_link">Technical Support</span>, or <span class="orderform_link">Professional Services</span>.
 
-      (b) Upon <span class="coverpage_link">Customer's</span> request, <span class="coverpage_link">Provider</span> will delete <span class="coverpage_link">Customer</span> Content within 60 days of termination.
+      (b) Upon <span class="keyterms_link">Customer's</span> request, <span class="keyterms_link">Provider</span> will delete Customer Content within 60 days of termination.
```

Comment:
Removing reference to termination because timing to perform is tied to the request, not date of termination

```suggestion
      (b) Upon <span class="keyterms_link">Customer's</span> request, <span class="keyterms_link">Provider</span> will delete Customer Content within 60 days.
```